events/class_tests/test_create_event_views.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from pytest_django.asserts import assertTemplateUsed
+from events.class_tests.create_events import DATE_TIME_START, DATE_TIME_END, COLOR, TITLE
```

Comment:
Please use only the conftest variables, the same applies to every line of code you write with constant variable that you imported or have written yourself.
if you have added a conftest file there is a better way to represent constant variable as you can see here:
https://github.com/redhat-beyond/lets-meet/blob/830988ce721fc82eafb792293b51bb2145b6de79/reminders/conftest.py#L15-L18

Reply:
Done